turbopack/crates/turbopack-node/src/transforms/webpack.rs

```diff
@@ -716,7 +716,7 @@ async fn dir_dependency_shallow(glob: Vc<ReadGlobResult>) -> Result<Vc<Completio
         // Reading all files to add itself as dependency
         match *item {
             DirectoryEntry::File(file) => {
-                file.track().await?;
+                file.read().await?;
```

Comment:
Some comment here re-iterating what you said in the PR summary would be useful, as this is unintuitive.